src/json.hpp.re2c

```diff
@@ -6168,6 +6168,21 @@ class basic_json
               thousands_sep(!loc->thousands_sep ? '\0' : loc->thousands_sep[0]),
               decimal_point(!loc->decimal_point ? '\0' : loc->decimal_point[0])
         {}
+        serializer(const serializer& cpy)
```

Comment:
We don't actually need to copy a serializer, so this should rather be

```cpp
        serializer(const serializer&) = delete;
        serializer& operator=(const serializer&) = delete;
```

to make GCC happy without adding unneeded features.